tests/uvdata/test_fhd.py

```diff
@@ -328,8 +328,9 @@ def test_read_fhd_latlonalt_match_xyz(fhd_data_files):
 
     mwa_tel = Telescope.from_known_telescopes("mwa")
 
+    # TODO: why don't these match better?
```

Comment:
I had the same question :laughing:

Reply:
I suspect this is due to some mismatch in where these files came from, but at this point it's unknowable without remaking the files.